compiler/il/AliasSetInterface.hpp

```diff
@@ -290,6 +300,10 @@ class TR_SymAliasSetInterface : public TR_AliasSetInterface<TR_SymAliasSetInterf
     static void setSymRef1KillsSymRef2Asymmetrically(TR::SymbolReference *symRef1, TR::SymbolReference *symRef2, bool includeGCSafePoint, bool value);
 
   TR::SymbolReference *_symbolReference;
+  //the node which initializes this class(if the class is initialized through a node)
+  TR::Node *_node;
+  //indicates if the class in initialized through a node.
+  bool from_node;
```

Comment:
If the intention of the comments is to document the fields we should use triple slashes so Doxygen will pick up the documentation.